S2VX.Game/Editor/ToolState/CameraToolState.cs

```diff
@@ -142,6 +142,12 @@ public class CameraToolState : S2VXToolState {
         public override bool OnToolKeyDown(KeyDownEvent e) {
             switch (e.Key) {
                 case Key.S:
+                    // Addresses https://github.com/maxrchung/S2VX/pull/341.
```

Comment:
Do we need to have this comment? We can always git blame to find the corresponding commit/issue

Reply:
That's true. The website has pretty good tools to view history too. I think this is a bit excessive for comments, so I'll remove issue references.